src/components/Requirement.vue

```diff
@@ -43,8 +51,8 @@ export default {
   data: function() {
     return {
       open: [],
-      viewDialog: false,
-      showDelete: false
+      hoveringOver: false,
+      iconHover: false,
```

Comment:
I've noticed that you started adopting my comma convention recently. It's nice for diffs!